packages/atomic/src/components/atomic-smart-snippet/atomic-smart-snippet-answer.tsx

```diff
@@ -32,6 +56,13 @@ export class AtomicSmartSnippetAnswer {
     return sanitizeStyle(this.innerStyle);
   }
 
+  private emitCurrentHeight() {
+    if (this.isRendering) {
```

Comment:
What is this check for (as well as the flag that accompanies it) ?

What would happen if it's removed ? Infinite rendering loop, or just some minor inefficient rendering loop ?

Reply:
Inefficient. I recall, when initially making the `collapsible-answer` component, that for a moment before `componentDidRender`, the element had the wrong height. This is to prevent that height from rendering first.